### DISCUSSION TITLE
Is it possible to add an index option for the public directory?

### DISCUSSION THREAD

**Discussion by p-chan** [vercel/next.js#15718] [Ideas]
I have a Next.js project at work that needs to host HTML. (Sometimes the outsourced company can't write React.)

So I decided to host the HTML files in the public directory, but there's one problem.

For example, I put the HTML file in `/public/example/index.html`.

I can access it at `http://locahost:3000/example/index.html`.
But if I go to `http://locahost:3000/example`, I get a 404  Not Found.

It would be better to be able to specify the index, like [expressjs/serve-static](https://github.com/expressjs/serve-static#index).

Please let me know what you think.

**Comment by matamatanot** [comment]
What about [Redirects](https://nextjs.org/docs/api-reference/next.config.js/redirects)?
